lib/settings/settingsPage.dart

```diff
@@ -0,0 +1,205 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
don't move settingsPage to this folder. We want to keep all main pages under lib